en/ros2/user_guide.md

```diff
@@ -464,6 +464,18 @@ At this point, every ROS2 node must be instructed to use the newly bridged `/clo
 This concludes the modifications required on the ROS2 side. On the PX4 side, you are only required to stop the uXRCE-DDS time synchronization, setting the parameter [UXRCE_DDS_SYNCT](../advanced_config/parameter_reference.md#UXRCE_DDS_SYNCT) to `false`.
 By doing so, Gazebo will act as main and only time source for both ROS2 and PX4.
 
+:::warning
```

Comment:
Thanks @federicociresola . 

How likely is this? My thinking is that if you follow the instructions, "not very", so perhaps this should move this down into the troubleshooting section?

@beniaminopozzan Are you happy with this?

Reply:
I think it is likely because when the official release of gazebo for ROS" humble is ignition not garden. You can look it [here ](https://gazebosim.org/docs/harmonic/ros_installation#summary-of-compatible-ros-and-gazebo-combinations)

Reply:
I agree that this might go in the troubleshooting section. There is already a (small) note about where to find the right version after all:
https://github.com/PX4/PX4-user_guide/blob/a79a24a30ad74150a3b7c27ca5f73e10b37a8cdf/en/ros2/user_guide.md?plain=1#L454

I think this is a good addition tot he doc as the existing note points to manual build installation instruction while users can actually install the right binaries.

@federicociresola , it seems to me that the package is `ros-humble-ros-gzgarden-bridge` though, could you double check? I'd also suggest to point to `ros-humble-ros-gzgarden`, so _all_ the ros/gz interface packages are installed, not just the bridge.

Reply:
@beniaminopozzan I didn't see the line but I think it is good to highlight it because a person can lose a lot of time trying to find the problem. Yes, you can suggest to install `ros-humble-ros-gzgarden-bridge` but I had a bad experience to install gazebo from ROS2 and gazebo standalone #23259. Then I think it is time to decide how to install gazebo (split in standalone or from ROS2 package) and should suggest a modification in `../setup/ubuntu.sh`. What do you think about it?

Reply:
@federicociresola Gazebo is installed via the script standalone because: 
- PX4 isn't validated with random versions, just Garden and Gazebo-Classic 11.
- Many (the majority) don't care about ROS.

Open to any solution that means you can still use PX4/Gazebo with and without ROS in a predictable way.

You haven't commented on my solution, which is to tell people exactly what they should install so there is no guesswork.

Reply:
you are right we can't modify the script because there is already the option to run the installation with/out the simulation tool.

Sorry, I have not commented on your solution because I have not seen it